README.md

```diff
@@ -288,6 +289,30 @@ Plugins specified in `requires` are removed when no active plugins require them.
 Plugins may be lazy-loaded on the use of keybindings/maps.
 Individual keybindings are specified either as a string (in which case they are treated as normal mode maps) or a table in the format `{mode, map}`.
 
+#### Custom loader
+
+A custom loader for a plugin may be specified via `cond`.
+This is a function which has a function as its first argument.
+When this function argument is called, the plugin is loaded.
+
+For example, the following plugin is lazy-loaded on the key mapping `ga`:
+
+```lua
+packer.add{
+  {"my/plugin", cond = function(load_plugin)
+    vim.keymap.set('n', 'ga', function()
+      vim.keymap.del('n', 'ga')
+      loader()
```

Comment:
Since the function argument is called `load_plugin` in the example, wouldn't that be `load_plugin()`?

```suggestion
      load_plugin()
```